jumanji/training/configs/env/bin_pack.yaml

```diff
@@ -8,20 +8,20 @@ network:
     transformer_mlp_units: [512]
 
 training:
-    num_epochs: 1000
+    num_epochs: 550
     num_learner_steps_per_epoch: 100
-    n_steps: 20
+    n_steps: 30
     total_batch_size: 64
 
 evaluation:
-    eval_total_batch_size: 5000
-    greedy_eval_total_batch_size: 5000
+    eval_total_batch_size: 10000
+    greedy_eval_total_batch_size: 10000
 
 a2c:
     normalize_advantage: False
     discount_factor: 1.0
     bootstrapping_factor: 0.95
     l_pg: 1.0
     l_td: 1.0
-    l_en: 0.01
+    l_en: 0.005
```

Comment:
Nit: would rename the loss coefficients to be more descriptive, but outside the scope of this MR

Reply:
At least we should have a readme :laughing: describing these arguments. I will make an issue for that.